### ISSUE TITLE
[BUG] `cyclonedx-python-lib` doesn't support wheel-only installation

### ISSUE THREAD

**Issue by madpah** [CycloneDX/cyclonedx-python-lib#97]
Source: https://github.com/sonatype-nexus-community/jake/issues/72

Validated by running:

```python
pip install cyclonedx-python-lib==0.12.2 --only-binary=:all:

Looking in indexes: https://OBFUSCATED/repository/pupy/simple
Collecting cyclonedx-python-lib==0.12.2
  Using cached https://OBFUSCATED/repository/pupy/packages/cyclonedx-python-lib/0.12.2/cyclonedx_python_lib-0.12.2-py3-none-any.whl (128 kB)
Collecting types-toml<0.11.0,>=0.10.0
  Using cached https://OBFUSCATED/repository/pupy/packages/types-toml/0.10.1/types_toml-0.10.1-py3-none-any.whl (2.1 kB)
Collecting packageurl-python>=0.9
  Using cached https://OBFUSCATED/repository/pupy/packages/packageurl-python/0.9.6/packageurl_python-0.9.6-py3-none-any.whl (23 kB)
ERROR: Could not find a version that satisfies the requirement requirements_parser>=0.2 (from cyclonedx-python-lib) (from versions: none)
ERROR: No matching distribution found for requirements_parser>=0.2
```

Note from https://pypi.org/project/requirements-parser/0.2.0/#files:
![Screenshot 2021-12-15 at 17 18 09](https://user-images.githubusercontent.com/10280392/146233812-df5a4394-17a4-4063-b8a2-7255703aa453.png)

**Comment by madpah** [comment]
I've reached out to the author of the `requirements-parser` project to see what options are possible.

**Comment by madpah** [comment]
Whilst we wait for the above to be resolved / migrated etc., we currently aren't using `requirements-parser` so we'll remove it. See #98

**Comment by madpah** [comment]
Give the PR is merged, we'll track anything further in #99.
